### PR TITLE
Template Dashboard (UI)

### PR DESCRIPTION
**Pull request by AlexWeinstein92**

### CODE REVIEW

File: ui/src/app/App.tsx [hunk 1]
```diff
@@ -39,6 +42,18 @@ function App() {
                                 <Route path="pricing">
                                     <Route index element={<Pricing/>}/>
                                 </Route>
+                                <Route path="dashboard">
+                                    <Route index element={<Navigate to="orders"/>}/>
```
[the remark this answers is not in the document]
Comment (AlexWeinstein92) [reply ui/src/app/App.tsx:46]: That's a decent idea, but I don't think it's necessary. I realized earlier that the selector for individual charts in a page ideally should be coupled with the charts themselves. So the pages may not use the URLs, but if someone wanted to go to a specific chart we could host them there as well.

**Review by miguelTaningcoYoppworks — APPROVED** [review]
Just a small nit if you want to apply it. Otherwise I don't see anything wrong with the PR assuming that it works fine.

File: ui/src/app/screens/styledComponents/navBars.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+import {useNavigate} from "react-router-dom";
+import MenuIcon from "@mui/icons-material/Menu";
+import Badge from "@mui/material/Badge";
+import NotificationsIcon from "@mui/icons-material/Notifications";
+import ChevronLeftIcon from "@mui/icons-material/ChevronLeft";
+import Divider from "@mui/material/Divider";
+import {mainNavItems, secondaryNavItems} from "../dashboard/navItems";
+import {Drawer} from "../dashboard/components";
+import List from "@mui/material/List";
+
```
Comment (miguelTaningcoYoppworks) [inline ui/src/app/screens/styledComponents/navBars.tsx:16]: The props on this page can be cleaned up by making an interface at the top of the file. Something like

```
interface SomeNameProps {
    forDashboard: boolean,
    children: ReactNode,
    ...
}
```

More of a suggestion than an order.

**Review by miguelTaningcoYoppworks — APPROVED** [review]
Looks good.

**Pull request merged**